tutorials/plot_parse_both.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+The Header Docstring
+====================
+
+When writting latex in a Python string keep in mind to escape the backslashes
+or use a raw docstring
```

Comment:
this file looks too much a copy paste from https://sphinx-gallery.github.io/tutorials/plot_parse.html
I would suggest to see how to integrate both so users find the information more easily.

my 2c

Reply:
It IS a copy paste from that. 
The original test I modified (`test_bug_cases_of_notebook_syntax()` from https://github.com/sphinx-gallery/sphinx-gallery/blob/master/sphinx_gallery/tests/test_gen_rst.py) tests that `split_code_and_text_blocks()` is performed correctly on the file 'tutorials/plot_parse.py'. I just amended that test to also test a version of 'plot_parse.py' ('plot_parse_both.py') which uses `# %%` as separators as well as `#`'s. 

Sorry, what do you mean integrate both? I can change the test to test only the 'new' plot_parse_both.py file which uses both `#`'s and `# %%` as a separator?

Reply:
now 2 almost identical files are going to show up on the documentation. I agree that it does the job for the testing but from a narrative doc perspective I would suggest this new example is as easy to discover as the old one OR to integrate them into one (at least for the documentation perspective).

my 2c

Reply:
Ah I see the problem now. I will integrate them into one.